app/Services/Forge/Api/SaloonForgeClient.php

```
@@ -0,0 +1,262 @@
+
+        return ForgeServerData::fromResource(JsonApiData::data($payload));
+    }
+
+    public function listSites(string|int $serverId): array
+    {
+        $resources = $this->paginate($this->endpoint("/servers/{$serverId}/sites"));
+
+        return array_map(static fn (array $resource) => ForgeSiteData::fromResource($resource), $resources);
+    }
+
+    public function getSite(string|int $serverId, string|int $siteId): ForgeSiteData
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$serverId'. (undefined)

(UnusedFormalParameter)